node/core/dispute-coordinator/src/tests.rs

```diff
@@ -2245,68 +2172,27 @@ fn resume_dispute_without_local_statement() {
 			)
 			.await;
 
-			let valid_vote0 = test_state
-				.issue_explicit_statement_with_index(
-					ValidatorIndex(0),
-					candidate_hash,
-					session,
-					true,
-				)
-				.await;
-			let valid_vote3 = test_state
-				.issue_explicit_statement_with_index(
-					ValidatorIndex(3),
-					candidate_hash,
-					session,
-					true,
-				)
-				.await;
-			let valid_vote4 = test_state
-				.issue_explicit_statement_with_index(
-					ValidatorIndex(4),
-					candidate_hash,
-					session,
-					true,
-				)
-				.await;
-			let valid_vote5 = test_state
-				.issue_explicit_statement_with_index(
-					ValidatorIndex(5),
-					candidate_hash,
-					session,
-					true,
-				)
-				.await;
-			let valid_vote6 = test_state
-				.issue_explicit_statement_with_index(
-					ValidatorIndex(6),
-					candidate_hash,
-					session,
-					true,
-				)
-				.await;
-			let valid_vote7 = test_state
-				.issue_explicit_statement_with_index(
-					ValidatorIndex(7),
-					candidate_hash,
-					session,
-					true,
-				)
-				.await;
+			let mut statements = Vec::new();
+			// Getting votes for supermajority. Should already have two valid votes.
+			for i in vec![3, 4, 5, 6, 7] {
```

Comment:
Late to the party here, but did you mean to leave out vote 0? (Nice PR BTW!)

Reply:
I did indeed mean to leave 0 out. It seems that 0 is our local vote. So in this test where we were supposed to trigger local participation we wouldn't need to submit a redundant explicit 0 vote.